File: Database/Models/Ingredient.cs
namespace Database.Models;

public class Ingredient
{
        public int Id { get; set; }
        public int SlvIngredientId { get; set; }
        public string Type { get; set; }
        public int Number { get; set; }
        public DateTime Version { get; set; }
        public string Name { get; set; }
        public IQueryable<ProductRecord> ProductRecords { get; set; }
}

File: FrontEnd_Daisy2/src/data/Ingredient.ts
export default interface Ingredient{
    Id: number;
    IngredientId: number;
    type: string;
    Number: number;
    Version: Date;
    Name: string;
}

export  interface IngredientDto{
    id: number;
    slvIngredientId: number;
    name: string;
}
